src/LibraryManager.Vsix/UI/Controls/TargetLocationAutomationPeer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Windows.Automation.Peers;
+using System.Windows.Controls;
+using Microsoft.Web.LibraryManager.Vsix.Resources;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI.Controls
+{
+    /// <summary>
+    /// Custom AutomationPeer for TargetLocation control
+    /// </summary>
+    internal class TargetLocationAutomationPeer : UserControlAutomationPeer
+    {
+        public TargetLocationAutomationPeer(UserControl owner) : base(owner)
+        {
+        }
+
+        protected override string GetLocalizedControlTypeCore()
```

Comment:
>GetLocalizedControlTypeCore [](start = 34, length = 27)

This should be returning the type of the control (e.g. Button or TextBox) rather than the specific label of the control. Instead of overriding this, you can override GetAutomationControlTypeCore() to return a value from the AutomationControlType enumeration, and base class implementation of GetLocalizedControlTypeCore will return a localized value.

More info:
[https://msdn.microsoft.com/en-us/library/system.windows.automation.peers.automationpeer.getlocalizedcontroltype(v=vs.110).aspx](https://msdn.microsoft.com/en-us/library/system.windows.automation.peers.automationpeer.getlocalizedcontroltype(v=vs.110).aspx)

Reply:
Updated in iteration 4.

---
In reply to: [210109772](https://github.com/aspnet/LibraryManager/pull/346#discussion_r210109772) [](ancestors = 210109772)